modules/mono/glue/GodotSharp/GodotSharp/Core/Vector4.cs

```diff
@@ -0,0 +1,730 @@
+        {
+            left.x *= right.x;
+            left.y *= right.y;
+            left.z *= right.z;
+            left.w *= right.w;
+            return left;
+        }
+
+        /// <summary>
+        /// Divides each component of the <see cref="Vector4"/>
+        /// by the given <see cref="real_t"/>.
+        /// </summary>
+        /// <param name="vec">The dividend vector.</param>
+        /// <param name="divisor">The divisor value.</param>
+        /// <returns>The divided vector.</returns>
+        public static Vector4 operator /(Vector4 vec, real_t divisor)
+        {
+            vec.x /= divisor;
+            vec.y /= divisor;
+            vec.z /= divisor;
+            vec.w /= divisor;
+            return vec;
+        }
+
+        /// <summary>
+        /// Divides each component of the <see cref="Vector4"/>
+        /// by the components of the given <see cref="Vector4"/>.
+        /// </summary>
+        /// <param name="vec">The dividend vector.</param>
+        /// <param name="divisorv">The divisor vector.</param>
+        /// <returns>The divided vector.</returns>
+        public static Vector4 operator /(Vector4 vec, Vector4 divisorv)
+        {
+            vec.x /= divisorv.x;
+            vec.y /= divisorv.y;
+            vec.z /= divisorv.z;
+            vec.w /= divisorv.w;
+            return vec;
+        }
+
+        /// <summary>
+        /// Returns <see langword="true"/> if the vectors are exactly equal.
+        /// Note: Due to floating-point precision errors, consider using
+        /// <see cref="IsEqualApprox"/> instead, which is more reliable.
+        /// </summary>
+        /// <param name="left">The left vector.</param>
+        /// <param name="right">The right vector.</param>
+        /// <returns>Whether or not the vectors are exactly equal.</returns>
+        public static bool operator ==(Vector4 left, Vector4 right)
+        {
+            return left.Equals(right);
+        }
+
+        /// <summary>
+        /// Returns <see langword="true"/> if the vectors are not equal.
+        /// Note: Due to floating-point precision errors, consider using
+        /// <see cref="IsEqualApprox"/> instead, which is more reliable.
+        /// </summary>
+        /// <param name="left">The left vector.</param>
+        /// <param name="right">The right vector.</param>
+        /// <returns>Whether or not the vectors are not equal.</returns>
+        public static bool operator !=(Vector4 left, Vector4 right)
+        {
+            return !left.Equals(right);
+        }
+
+        /// <summary>
+        /// Compares two <see cref="Vector4"/> vectors by first checking if
+        /// the X value of the <paramref name="left"/> vector is less than
+        /// the X value of the <paramref name="right"/> vector.
+        /// If the X values are exactly equal, then it repeats this check
+        /// with the Y values of the two vectors, and then with the Z values.
```

Comment:
```suggestion
        /// with the Y values of the two vectors, then with the Z values, and then with the W values.
```

Reply:
With the longer text, we should also give it another line.

Reply:
I think adding W is pretty long,
I would prefer a shorter version
- If the X values are exactly equal, then it repeats this check with the Y, Z and W values in order.
- If the X values are exactly equal, then it repeats this check with the Y, Z and finally W values of the two vectors.